packages/node_modules/@cerebral/firebase/README.md

```diff
@@ -260,37 +343,33 @@ Note: When using transactions with Security and Firebase Rules in place, be awar
 
 ## value
 
-*action*
+### action
 ```js
 function someAction({ firebase }) {
-  return firebase.value('someKey.foo', {
```

Comment:
@christianalfoni I have a feeling that this part should be kept in some way. Could you confirm that?